tfx/experimental/distributed_inference/graphdef_experiments/subgraph_partitioning/create_complex_graph.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generate an example consists of {'main', 'remote_op_a', 'remote_op_b'}.
+
+When calling a remote op, we feed in some inputs, execute the graph associated
+  with the remote op, and get an output.
+
+In this example, main calls each of remote_op_a and remote_op_b two times,
+  remote_op_b calls remote_op_a two times, and remote_op_a calls no one.
+"""
+
+import tensorflow as tf
+
+
+tf.compat.v1.disable_eager_execution()  # Disable eager mode
+
+N = 1000  # number of embeddings
+NDIMS = 16  # dimensionality of embeddings
+
+
+def create_session(graph):
+  return tf.compat.v1.Session(graph=graph,
+                              config=tf.compat.v1.ConfigProto(
+                                  inter_op_parallelism_threads=8))
+
+
+# Define remote_op_a's graph
+graph_a = tf.Graph()
+with graph_a.as_default():
+  table_a = tf.random.uniform(shape=[N, NDIMS], seed=10)
+  ids_a = tf.compat.v1.placeholder(dtype=tf.int32, name='ids_a')
+  result_a = tf.nn.embedding_lookup(table_a, ids_a)
+
+
+def remote_op_a(input_ids):
+  """Mimic a remote op by numpy_function"""
+  def remote_loopup(input_ids):
```

Comment:
`remote_loopup` -> `remote_lookup` here and below

Reply:
Changed.